ServoConfig config
---
bool success
string extra_info

